schema.sql

```diff
@@ -35,4 +35,43 @@ ALTER TABLE animals ADD CONSTRAINT fk_species_id FOREIGN KEY (species_id) REFERE
 ALTER TABLE animals ADD COLUMN owner_id INT;
 ALTER TABLE animals ADD CONSTRAINT fk_owner_id FOREIGN KEY (owner_id) REFERENCES owners (id);
 
+-- Create vets table
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250) NOT NULL,
+    age INT NOT NULL,
+    date_of_graduation DATE NULL,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE vets (id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250) NOT NULL,
+    age INT NOT NULL,
+    date_of_graduation DATE NULL,
+    PRIMARY KEY(id)
+);
```

Comment:
- [ ] There are two commands to create the vets table. Please kindly remove one

Reply:
well resolved! ✔️